---
layout: archive
title: "Process"
permalink: /process/
author_profile: true

---

{% include base_path %}

Education
======

* B.S. in Shandong, Shandong University, 2026(excepted)

Research Experiebcence
======
* Detection of RNA Modifications using Nanopore Direct RNA Sequencing Reads
* Jun. 2023 – Present:Research Assistant, Research Center of Software and Data Engineering
  * Processed raw Nanopore Fast5 files, including basecalling, resquiggle operations, and signal-level analysis. Extracted crucial information for RNA modification detection through read alignment and signal segmentation
  * Conducted site-level analysis of RNA methylation sequencing data, focusing on accurate detection of methylation sites.
  * Applied deep learning multimodal models to enhance RNA methylation prediction accuracy, contributing to advancements in RNA modification research.

* Prediction of EGFR Mutations in Lung Cancer using Multi-Modal Patient Data
* Jul. 2024 –Present:Research Assistant, Advanced Medical Research Institute
  * Collected and processed multi-modal hospital data, including blood test results, CT imaging, and EGFR mutation test results, for use in predictive modeling.
  * Developed and trained ensemble learning models to predict EGFR mutations, utilizing algorithms such as Support Vector Machines (SVM), Random Forest, and Gradient Descent.
  * Conducted rigorous model evaluation and validation, applying cross-validation techniques and performance metrics to ensure robust and clinically relevant predictions.
  * Collaborated with clinical teams to ensure models could be effectively integrated into healthcare settings.

* The Impact of Microgravity on the Interaction Between Organs
* Jul. 2024 –Present: Research Assistant, Advanced Medical Research Institute
  * Collected and processed RNA-seq and single-cell data, performed cohort grouping, and eliminated batch effects for accurate comparative analysis.
  * Applied machine learning techniques to rank gene impacts and visualized differential gene expression between ground control (GC) and spaceflight (FLT) groups, identifying shared patterns of gene regulation across multiple organs.
  * Conducted pathway analysis to identify critical pathways influenced by microgravity, uncovering key biological mechanisms affected across organs.
  * Integrated multi-omics data to assess specific organ-level effects of microgravity, validating key findings through experimental data.

* Integration of Spatial Transcriptomics and Single-Cell Sequencing Analysis 
* Jul. 2024 –Present: Research Assistant, Advanced Medical Research Institute
  * Performed deconvolution analysis on spatial transcriptomics data, revealing cell distribution and gene expression patterns across different spatial regions.
  * Utilized Python libraries like Scanpy for clustering and classification of single-cell data, annotating cell types through comprehensive literature review and database research.
  * Refined spatial transcriptomics data by aligning single-cell classifications with spatially resolved gene expression, providing detailed insights into tissue composition.
  * Integrated spatial transcriptomics with single-cell sequencing to investigate cell communication, developmental trajectories, and other high-resolution cellular interactions.


Skills
======
* Machine Learning
* Code
  * Python
  * C++
  * Java
* Single-cell transcriptomics data analysis
* Spatial transcriptomics data analysis
  
Service and leadership
======
* Currently signed in to 43 different slack teams
